python-avd/pyavd/api/ip_addressing/__init__.py

```diff
@@ -314,16 +314,11 @@ def vrf_loopback_ip(self, pool: str) -> str:
     def evpn_underlay_l3_multicast_group(
         self,
         underlay_l3_multicast_group_ipv4_pool: str,
-        vrf_vni: int,
-        vrf_id: int,  # pylint: disable=unused-argument # noqa: ARG002
+        vrf_id: int,
```

Comment:
Please do not change the signature of the function. These ip_addressing methods can be overridden by expert users with custom Python modules. Just move the pylint/noqa to the vrf_vni field.

Reply:
Fixed!

Reply:
sonarcloud is failing after including the variable, can we move it to Accepted issues?

Reply:
I have accepted the issue.